examples/devconsole_v1alpha1_component_cr.yaml

```diff
@@ -5,4 +5,6 @@ metadata:
 spec:
   buildType: "nodejs"
   codebase: "https://github.com/sclorg/nodejs-ex" #"https://github.com/nodeshift-starters/nodejs-rest-http-crud"
+  port: "8080"
```

Comment:
we are using `8080` as default, maybe we can some different value, to not confuse user who is reading this example

Reply:
ok, changed by 3000